sys/include/net/gnrc/nettype.h

```diff
@@ -120,6 +120,15 @@ typedef enum {
 #endif
     /** @} */
 
+    /**
+     * @{
+     * @name Protocol-independent network events.
+     * }
+     */
+    GNRC_NETTYPE_L2_DISCOVERY,  /**< Layer 2 node discovery */
+    GNRC_NETTYPE_L3_ROUTING,    /**< Layer 3 routing info */
```

Comment:
What does 'routing info' mean? When will this event be generated - when a route changes?